sale_timesheet_task_exclude/tests/test_sale_timesheet_exclude_task.py

```diff
@@ -116,15 +116,12 @@ def test_exclude_from_sale_order(self):
                 "employee_id": employee.id,
             }
         )
-        self.assertTrue(timesheet.so_line)
-        self.assertEqual(task.billable_type, "task_rate")
-
         task.exclude_from_sale_order = True
-        self.assertEqual(task.billable_type, "no")
+        task.allow_billable = False
```

Comment:
Setting `exclude_from_sale_order` to true is not enough to ensure `so_line` being False?

Reply:
No we require to set allow_billable to False